packages/swayze/lib/src/widgets/headers/gestures/header_gesture_detector.dart

```diff
@@ -250,67 +273,216 @@ class _HeaderGestureDetectorState extends State<HeaderGestureDetector> {
     );
   }
 
+  /// Handles drag starts that should start dragging a header around.
+  void handleStartDraggingHeader(
+    DragStartDetails details,
+    Range selectionRange,
+  ) {
+    setCursorState(SystemMouseCursors.basic);
```

Comment:
For now, I set the cursor back to its basic state when the header is being dragged.

This happens because there is no way to change the cursor globally, and it only affects a certain `MouseRegion`, so the closed hand cursor would be showing only when the mouse is hovering over the headers.

Reply:
Would it make sense to have this explanation as a comment in the code? I think it would help future developers understand why we're doing it like this. 

Also, is there any Flutter issue we can track for this?